### DISCUSSION TITLE
Month localization in date

### DISCUSSION THREAD

**Discussion by mcg-pimcore** [pimcore/pimcore#10188] [Q&A / Support] [answered]
In pimcore do we have option to localize date month.

**Comment by aarongerig** [comment, marked as answer]
Hi @mcg-pimcore ,

Yes, have a look into Carbon Date: https://carbon.nesbot.com/docs/

**Reply by mcg-pimcore** [reply]
Thanks for reply.

**Reply by mcg-pimcore** [reply]
use Carbon\Carbon;
​
$date = Carbon::now()->locale('fr_FR');

echo $date->locale(); // fr_FR
echo "\n";
echo $date->diffForHumans(); // il y a 1 seconde
echo "\n";
echo $date->monthName; // juillet
echo "\n";
echo $date->isoFormat('LLLL'); // jeudi 1 juillet 2021 08:55
